app/decorators/paginating_decorator.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class PaginatingDecorator < Draper::CollectionDecorator
+  # support for will_paginate
+  delegate :current_page, :total_entries, :total_pages, :per_page, :offset, :limit_value, :model_name, :total_count
```

Comment:
https://github.com/3scale/porta/pull/2099#discussion_r464959448

Reply:
Or we can add the method ourselves without the gem. It's only one method that needs to be patched (`CanCan::Ability#can?`) after all, and whose implementation can be improved BTW.